### PR TITLE
Add support for playing a "LoseNotification" to the old owner through CaptureNotification

### PR DESCRIPTION
**Pull request by abcdefg30**
We want this for at least ra2: https://github.com/OpenRA/ra2/pull/187#discussion_r59159963

The default being `null` won't cause trouble as `PlayNotification` already catches that.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/Sound/CaptureNotification.cs [hunk 1]
```diff
@@ -15,27 +15,36 @@ namespace OpenRA.Mods.Common.Traits.Sound
...
 		public void OnCapture(Actor self, Actor captor, Player oldOwner, Player newOwner)
 		{
-			if (captor.World.LocalPlayer != captor.Owner)
-				return;
-
 			var faction = info.NewOwnerVoice ? newOwner.Faction.InternalName : oldOwner.Faction.InternalName;
-			Game.Sound.PlayNotification(self.World.Map.Rules, captor.World.LocalPlayer, "Speech", info.Notification, faction);
+			Game.Sound.PlayNotification(self.World.Map.Rules, newOwner, "Speech", info.Notification, faction);
+
+			var looseFaction = info.NewOwnerVoice ? newOwner.Faction.InternalName : oldOwner.Faction.InternalName;
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/Sound/CaptureNotification.cs]: This wants to be `info.LooseNewOwnerVoice` I assume.

Comment (abcdefg30) [inline OpenRA.Mods.Common/Traits/Sound/CaptureNotification.cs]: Derp, right.

**Comment by abcdefg30** [comment]
Fixed.

**Comment by Phrohdoh** [comment]
I think you guys want "Lose" not "Loose".

>  adjective, looser, loosest.
free or released from fastening or attachment:
a loose end.

**Comment by abcdefg30** [comment]
Yeah, already fixed that, just not the descriptions and commit messages...

**Comment by reaperrr** [comment]
:+1:

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/00693c94d7b29afd998ed86875162239eec715e3...3335c028ca798aea033736cbe72248bf14fad3fd)

**Pull request merged**